config/govuk_index/migrated_formats.yaml

```diff
@@ -48,6 +48,8 @@ migrated:
 - statutory_instrument
 - step_by_step_nav
 - task_list
+- news_story: # not indexable when published by Whitehall
+  - content-publisher
```

Comment:
maybe we could have this as an object of `publishing_app: content-publisher` as this might make the intent somewhat clearer?

Reply:
I've updated https://github.com/alphagov/rummager/pull/1276/commits/a58e8e6f4325494ec88e4244f785f6b83228b5cd with this new data format.